tests/test_snappy_pull_sheets.py

```diff
@@ -64,57 +63,78 @@ def return_api_investigation_mock():
     return investigation
 
 
-@pytest.fixture
-def pull_sheet_config():
-    """Returns empty PullSheetsConfig object"""
-    global_config = CommonConfig(
-        **{"verbose": False, "sodar_api_token": "__secret__", "sodar_server_url": "url"}
-    )
-    args = {
-        "global_config": global_config,
-        "base_path": None,
-        "yes": False,
-        "dry_run": False,
-        "show_diff": False,
-        "show_diff_side_by_side": False,
-        "library_types": ("WES", "RNA_seq"),
-        "first_batch": 0,
-        "last_batch": None,
-        "tsv_shortcut": "germline",
-        "assay_uuid": None
-    }
-    return PullSheetsConfig(**args)
-
 
-def test_build_sheet_germline(mocker, pull_sheet_config):
+def test_build_sheet_germline(mocker):
     """Tests ``build_sheet()`` - for germline ISA tab"""
+    args = Namespace( verbose = False,
+            sodar_api_token= "__secret__",
+            sodar_server_url= "url",
+            base_path= None,
+            yes= False,
+            dry_run= False,
+            show_diff= False,
+            show_diff_side_by_side= False,
+            library_types= ("WES", "RNA_seq"),
+            first_batch= 0,
+            last_batch= None,
+            tsv_shortcut= "germline",
+            assay_uuid= None
+    )
     path = pathlib.Path(__file__).resolve().parent / "data" / "pull_sheets" / "sheet_germline.tsv"
     with open(path, "r") as file:
         expected = "".join(file.readlines())
     mocker.patch(
         "sodar_cli.api.samplesheet.export", return_value=load_isa_dict("isa_dict_germline.txt")
     )
-    actual = build_sheet(config=pull_sheet_config, project_uuid="")
+    actual = build_sheet(args=args, project_uuid="")
     assert actual == expected
 
 
-def test_build_sheet_cancer(mocker, pull_sheet_config):
+def test_build_sheet_cancer(mocker):
     """Tests ``build_sheet()`` - for cancer ISA tab"""
+    args = Namespace( verbose = False,
+            sodar_api_token= "__secret__",
+            sodar_server_url= "url",
+            base_path= None,
+            yes= False,
+            dry_run= False,
+            show_diff= False,
+            show_diff_side_by_side= False,
+            library_types= ("WES", "RNA_seq"),
+            first_batch= 0,
+            last_batch= None,
+            tsv_shortcut= "cancer",
+            assay_uuid= None
+    )
     path = pathlib.Path(__file__).resolve().parent / "data" / "pull_sheets" / "sheet_cancer.tsv"
     with open(path, "r") as file:
         expected = "".join(file.readlines())
     mocker.patch(
         "sodar_cli.api.samplesheet.export", return_value=load_isa_dict("isa_dict_cancer.txt")
     )
```

Comment:
Why not use the `@patch` decorator here? (from unittest.mock import patch)
I guess either way is fine. Some of the (newer) tests inc ubi-tk use the decorator approach, the older ones use various mocker approaches (often without using the mocker fixture though), so we should decide on one approach to go with.